docs/en/latest/terminology/plugin-config.md

```diff
@@ -30,7 +30,7 @@ description: Plugin Config in Apache APISIX.
 
 Plugin Configs are used to extract commonly used [Plugin](./plugin.md) configurations and can be bound directly to a [Route](./route.md).
 
-While configuring the same plugin, only one copy of the configuration is valid. The order of precedence is always `Consumer` > `Consumer Group` > `Route` > `Plugin Config` > `Service`.
+While configuring the same plugin, only one copy of the configuration is valid. The order of precedence is always `Consumer` > `Consumer Group` > `Route` > `Plugin Config` > `Service`. Plugins configured in `Global Rules` will be executed first at all times.
```

Comment:
I suggest to leverage these information when it comes to plugin execution orders and precedence, to avoid update anomalies. 

* https://apisix.apache.org/docs/apisix/next/terminology/plugin/#plugins-merging-precedence
* https://apisix.apache.org/docs/apisix/next/terminology/plugin/#plugins-execution-order